script.js

```diff
@@ -20,3 +22,144 @@ openBtn.addEventListener("click", openElements);
+    technologies: ["Ruby", "HTML", "CSS", "Javascript"],
+    image: "./img/design-firm.jpg",
+    link: "https://serengia.github.io/Portfolio-setup-and-mobile-first/",
+    linkSource: "https://github.com/Serengia/Portfolio-setup-and-mobile-first",
+  },
+  {
+    id: "4",
+    name: "Logistics Web Application",
+    technologies: ["Ruby", "HTML", "CSS", "Javascript"],
+    image: "./img/logistics.jpg",
+    link: "https://serengia.github.io/Portfolio-setup-and-mobile-first/",
+    linkSource: "https://github.com/Serengia/Portfolio-setup-and-mobile-first",
+  },
+  {
+    id: "5",
+    name: "Saas Legacy Code Update",
+    technologies: ["Ruby", "HTML", "CSS", "Javascript"],
+    image: "./img/legacy-code-update.jpg",
+    link: "https://serengia.github.io/Portfolio-setup-and-mobile-first/",
+    linkSource: "https://github.com/Serengia/Portfolio-setup-and-mobile-first",
+  },
+  {
+    id: "6",
+    name: "School Software System",
+    technologies: ["Ruby", "Javascript"],
+    image: "./img/web-development.jpg",
+    link: "https://serengia.github.io/Portfolio-setup-and-mobile-first/",
+    linkSource: "https://github.com/Serengia/Portfolio-setup-and-mobile-first",
+  },
+];
+
+const generateModalMarkup = (data) => {
+  const listMarkup = data.technologies
+    .map((item) => `<li class="skill-tag skill-tag--grey">${item}</li>`)
+    .join("");
+
+  const modalMarkup = ` 
+  <div class="backdrop">
+
+  <div class="modal">
+    <svg class="modal-close-icon" width="24" height="24" viewBox="0 0 24 24" fill="none"
+      xmlns="http://www.w3.org/2000/svg">
+      <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+         />
+    </svg>
+    <img class="modal-img" src="${data.image}" alt="Modal desktop">
+   
+      <h2>${data.name}</h2>
+      <div class="button-container ">
+        <a href="${data.link}">See Live <img src="./img/modal-live-icon.svg" alt="Modal icon"></a>
+        <a href="${data.linkSource}">See source <img src="./img/modal-gihub-icon.svg" alt="Modal icon"> </a>
+      </div>
+   
+    <ul class="modal-skills">
+      ${listMarkup}
+    </ul>
+    <p class="modal-description">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Officia, saepe
+      voluptatibus. Nihil exercitationem facilis, eligendi!
+    </p>
+
+  </div>
+</div>`;
+
+  return modalMarkup;
+};
+
+const openModal = (id) => {
+  const activeModalData = projectData.find((el) => el.id === `${id}`);
+  console.log(activeModalData);
```

Comment:
- [ optional ] As part of JS best practice, you are required to keep your code clean. So I recommend removing the unnecessary `console.log` in your JS file